### PR TITLE
adding missing tests

### PR DESCRIPTION
**Pull request by Tokesh**
### Description
 DELETE /_search/scroll
 POST /_snapshot/{repository}/{snapshot}
 POST /_update_by_query/{task_id}/_rethrottle
 PUT /_wlm/query_group/{name}

### Issues Resolved
[#663 ]

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: tests/default/_core/update_by_query.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+$schema: ../../../json_schemas/test_story.schema.yaml
```
Comment (dblock) [inline tests/default/_core/update_by_query.yaml:1]: This is _update_by_query/rethrottle, so it should be in `tests/default/_core/_update_by_query/rethrottle.yaml`.

File: tests/default/_core/update_by_query.yaml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+description: Test Update By Query endpoint.
```
Comment (dblock) [inline tests/default/_core/update_by_query.yaml:3]:
```suggestion
description: Test Update By Query rethrottle endpoint.
```

File: tests/default/_core/update_by_query.yaml [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+prologues:
+  - path: /books/_delete_by_query
```
Comment (dblock) [inline tests/default/_core/update_by_query.yaml:5]: Should this be trying to do an `update_by_query` and not a `_delete_by_query`?

File: tests/snapshot/snapshot/snapshot.yaml [hunk 4]
```diff
@@ -6,21 +6,27 @@ epilogues:
     method: DELETE
     status: [200, 404]
     parameters:
-      repository: my-fs-repository
```
Comment (dblock) [inline tests/snapshot/snapshot/snapshot.yaml:9]: We have a bunch of tests where we say `my-fs-repository`, either change them all or change this back to `my-fs-repository`.

**Review by dblock — CHANGES_REQUESTED** [review]
Some asks below.

Throw a retry into the flaky snapshot test.

**Comment by Tokesh** [comment]
> Some asks below.
> 
> Throw a retry into the flaky snapshot test.

Thank you! Ready for your review

**Review by dblock — APPROVED** [review]

**Pull request merged**